README.md

```diff
@@ -0,0 +1,87 @@
+![Aptabase](https://aptabase.com/og.png)
+
+# Unreal Engine SDK for Aptabase
+
+Instrument your Unreal Engine project with Aptabase, an Open Source, Privacy-First and Simple Analytics for Mobile, Desktop and Web Apps.
+
+## Install
+
+Install the SDK via the [Unreal Marketplace](TODO) or clone this repository inside your `Plugins` folder and compile it (requires C++ based project).
+
+## Enabling
+
+The first thing you will need to do is to enable the plugin in your project.
+
+1. Navigate to `Toolbar menu → Edit → Plugins` and open the `Plugins` tab.
+1. Search for `Aptabase`.
+1. Enable the `Aptabase` plugin.
+1. Press on the `Restart now` pop-up button or restart your project manually.
+
+## Provider setup
+
+Inside the `Config/DefaultEngine.ini` file add the following lines:
+
+```
+[Analytics]
+ProviderModuleName=Aptabase
+```
+
+**Note**: If you already have a different Analytics provider, you will have to replace it with `Aptabase` or use the [Multicast Analytics Provider Plugin](https://docs.unrealengine.com/4.26/en-US/TestingAndOptimization/Analytics/Multicast/) to run multiple in parallel.
+
+## Project settings
+
+First, you need to get your `App Key` from Aptabase, you can find it in the `Instructions` menu on the left side menu.
+
+Then you have to set it inside `Project Settings → Analytics → Aptabase` inside your `Api Key` field.
```

Comment:
And App Key here as well

Reply:
Everything has been renamed to `AppKey`.